products/bootstrap.md

```diff
@@ -70,5 +75,6 @@ Versions in _Maintenance_ should not have any changes landed, except for:
 - **Important** documentation updates
 
 Unless a change is urgent, _Maintenance_ releases are likely to be made with minimal frequency.
-The `v4` branch is currently in Active LTS and will receive bug fixes till 2021-07-01, after which
-it will only receive critical fixes till its End of Life.
+Maintenance status will expire on the dates listed in the table above (middle column) and the version will reach End of Life, where no more updates will be published.
+
+If you are still using Bootstrap 3 or 4, which have both reached end of life, commercial support for those versions is available through the [HeroDevs Never-Ending Support](https://www.herodevs.com/support/nes-bootstrap) initiative. For more information, see Bootstrap [3.4.1](https://getbootstrap.com/docs/3.4/) and [4.6.x](https://getbootstrap.com/docs/4.6/getting-started/introduction/) docs.
```

Comment:
```suggestion
Commercial support is available for some deprecated versions of Bootstrap through the [HeroDevs Never-Ending Support](https://www.herodevs.com/support/nes-bootstrap) initiative.
```

Changed the sentence to a more neutral/informative form, similar to https://endoflife.date/angular. Also removed the last sentence (`For more information...`) because it just links to https://www.herodevs.com/support/nes-bootstrap ultimately, which contains all the necessary information.

Reply:
I updated the paragraph with the more neutral/informative form. 

However, we'd like to keep the specific terminology 'end of life' versus 'deprecated' if feasible, as EOL has specific meaning to Bootstrap and HeroDevs. Also, we're hoping to keep in the final 'For more information' sentence. I changed the links in that sentence to clearly point to the specific EOL pages in the Bootstrap docs though. We think it might be helpful for users unfamiliar with HeroDevs to see version specific documentation from Bootstrap in order to decide how they'd like to proceed with their EOL package.

Reply:
Ok, thanks for the update.